include/ofi_hmem.h

```diff
@@ -253,9 +257,10 @@ ssize_t ofi_copy_to_hmem_iov(enum fi_hmem_iface hmem_iface, uint64_t device,
 			     size_t hmem_iov_count, uint64_t hmem_iov_offset,
 			     const void *src, size_t size);
 
-int ofi_hmem_get_handle(enum fi_hmem_iface iface, void *dev_buf, void **handle);
+int ofi_hmem_get_handle(enum fi_hmem_iface iface, void *dev_buf,
```

Comment:
I do not think you should modify this API. I guess you may want to deal with the length of the buffer allocation and the offset of the buffer that is related to the IPC handle. You can use the `ofi_hmem_get_base_addr` to get the base address of the allocation and then do the offset math.